lib/Sema/NameBinding.cpp

```diff
@@ -978,6 +1017,11 @@ void NameBinder::findCrossImports(UnboundImport &I,
     unboundImports.emplace_back(declaringImport.module.second->getASTContext(),
                                 I, name, declaringImport, bystandingImport);
 
+    if (llvm::is_contained(oppositeNames, name))
```

Comment:
I think we should default-initialize `oppositeNames` instead of leaving it uninitialized; otherwise, we might access uninitialized memory here?

Reply:
A `SmallVector` is default-initialized without doing anything special—only primitive types (including pointers) are left uninitialized by default in C++.

Reply:
Whaaaaat, that is so weird. I would've though leaving larger types uninitialized would be more useful than smaller types because there is more work that can be avoided. Sigh.